game/sony/PS/PSP/psp_save_metadata.ksy

```diff
@@ -0,0 +1,102 @@
+meta:
+  id: psp_save_metadata
```

Comment:
File name:

* We don't really have any decent support for nested directories yet
* There is little value in starting system of directories (i.e. `sony`, `PS`, `PSP`) when there's no other files to put there
* I don't see why we should use upper case in dir names here
* Compile results are 100% flat, so you'd probably like to include stuff that removes ambiguolity it into file name anyway, i.e. using full names instead of abbreviations, like `sony_playstation_portable_save_metadata` or at least `sony_psp_save_metadata`. `psp` is [way too ambiguous](https://en.wikipedia.org/wiki/PSP), even just in "Computing".

So, basically, my suggestion: let's use `game/sony_playstation_portable_save_metadat` or `game/sony_psp_save_metadata` for now?